lib/ui/main.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_contact_list/ui/contact-list.dart';
```

Comment:
1: File name should be 'contact_list_screen.dart':
https://dart.dev/guides/language/effective-dart/style
2: Should move the 'full screen' widgets to a sub folder 'screens' (/ui/screens)

Reply:
Changed the name of file for better style